### ISSUE TITLE
feature request: include doc comments in embed

### ISSUE THREAD

**Issue by liamaharon** [sam0x17/docify#8]
It would be helpful if the doc comment for exported items could be optionally included with the code.

**Comment by sam0x17** [comment]
Yes I was actually thinking about this when I saw your previous issue. I'll see if I can work it in soon

**Comment by kianenigma** [comment]
Ideally, I would expand this to any attribute. `#[doc = ".."]` is then merely one specialized case for this. 

So, we need a way to say: 
1. bring in all attributes (default behavior) 
2. don't bring in any attributes, 
3. bring in only these attributes 
4. bring in all attributes except these.

**Comment by sam0x17** [comment]
if we want to get that granular, might make sense to simply have an additional attribute `#[docify::capture_start]` and `#[docify::capture_end]` which will capture any attributes appearing between those two and also include them in the export

**Comment by kianenigma** [comment]
Simplest use case is that atm 

```rust
#[docify::export]
/// A mapping from `T::AccountId` to `Balance`.
#[pallet::storage]
pub type Balances<T: Config> = StorageMap<Key = T::AccountId, Value = Balance>;
```

is not capable of picking up the docs, and it would be good if it does. No strong opinion about the syntax.
